<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>778</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" />
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>778</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew" />
    <addaction name="actionOpen" />
    <addaction name="actionSave" />
    <addaction name="actionSave_As" />
    <addaction name="actionClose" />
    <addaction name="separator" />
    <addaction name="actionQuit" />
   </widget>
   <widget class="QMenu" name="menuEdit" >
    <property name="title" >
     <string>Edit</string>
    </property>
    <addaction name="actionUndo" />
    <addaction name="actionRedo" />
    <addaction name="separator" />
    <addaction name="actionCopy" />
    <addaction name="actionCut" />
    <addaction name="actionPaste" />
    <addaction name="separator" />
    <addaction name="actionPersisten_selection_mode" />
    <addaction name="actionMultiline_edit_mode" />
    <addaction name="separator" />
    <addaction name="actionSelect_All" />
   </widget>
   <widget class="QMenu" name="menuHelp" >
    <property name="title" >
     <string>Help</string>
    </property>
    <addaction name="action_About" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuEdit" />
   <addaction name="menuHelp" />
  </widget>
  <widget class="QStatusBar" name="statusbar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>581</y>
     <width>778</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar" >
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea" >
    <number>4</number>
   </attribute>
   <addaction name="actionNew" />
   <addaction name="actionOpen" />
   <addaction name="actionSave" />
   <addaction name="separator" />
   <addaction name="actionUndo" />
   <addaction name="actionRedo" />
   <addaction name="separator" />
   <addaction name="actionCopy" />
   <addaction name="actionCut" />
   <addaction name="actionPaste" />
   <addaction name="separator" />
   <addaction name="actionPersisten_selection_mode" />
   <addaction name="actionMultiline_edit_mode" />
  </widget>
  <action name="actionNew" >
   <property name="icon" >
    <iconset resource="..\sphere.qrc" >:/icons/images/new.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;New</string>
   </property>
  </action>
  <action name="actionOpen" >
   <property name="icon" >
    <iconset resource="..\sphere.qrc" >:/icons/images/open.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="actionSave" >
   <property name="icon" >
    <iconset resource="..\sphere.qrc" >:/icons/images/save.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionSave_As" >
   <property name="icon" >
    <iconset resource="..\sphere.qrc" >:/icons/images/saveas.png</iconset>
   </property>
   <property name="text" >
    <string>Save &amp;As...</string>
   </property>
  </action>
  <action name="actionClose" >
   <property name="text" >
    <string>&amp;Close</string>
   </property>
  </action>
  <action name="actionQuit" >
   <property name="text" >
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionUndo" >
   <property name="icon" >
    <iconset resource="..\sphere.qrc" >:/icons/images/undo.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Undo</string>
   </property>
  </action>
  <action name="actionRedo" >
   <property name="icon" >
    <iconset resource="..\sphere.qrc" >:/icons/images/redo.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Redo</string>
   </property>
  </action>
  <action name="actionCopy" >
   <property name="icon" >
    <iconset resource="..\sphere.qrc" >:/icons/images/copy.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Copy</string>
   </property>
  </action>
  <action name="actionCut" >
   <property name="icon" >
    <iconset resource="..\sphere.qrc" >:/icons/images/cut.png</iconset>
   </property>
   <property name="text" >
    <string>Cu&amp;t</string>
   </property>
  </action>
  <action name="actionPaste" >
   <property name="icon" >
    <iconset resource="..\sphere.qrc" >:/icons/images/paste.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Paste</string>
   </property>
  </action>
  <action name="actionSelect_All" >
   <property name="text" >
    <string>Select &amp;All</string>
   </property>
  </action>
  <action name="action_About" >
   <property name="text" >
    <string>&amp;About...</string>
   </property>
  </action>
  <action name="actionPersisten_selection_mode" >
   <property name="icon" >
    <iconset resource="..\sphere.qrc" >:/icons/images/persistentselection.png</iconset>
   </property>
   <property name="text" >
    <string>Persisten selection mode</string>
   </property>
  </action>
  <action name="actionMultiline_edit_mode" >
   <property name="icon" >
    <iconset resource="..\sphere.qrc" >:/icons/images/multilineedit.png</iconset>
   </property>
   <property name="text" >
    <string>Multiline edit mode</string>
   </property>
  </action>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources>
  <include location="../sphere.qrc" />
 </resources>
 <connections/>
</ui>
